windows/src/engine/kmtip/keys.cpp

```diff
@@ -168,7 +174,8 @@ STDAPI CKMTipTextService::OnTestKeyDown(ITfContext *pContext, WPARAM wParam, LPA
 {
   LogKey("CKMTipTextService::OnTestKeyDown", 0, wParam, lParam);
   // If the keystroke is a Keyman-generated key, ignore it
-  if((lParam & 0x00FF0000L) == 0xFF0000L)   // I3566
+  if((lParam & 0x00FF0000L) == 0xFF0000L &&
+    wParam != VK_CAPITAL)   // I3566
```

Comment:
We always need to pass Caps Lock through to the engine for processing, even if we generated it (so we track the state reliably)